scripts/data/summeval_fr/create_data.py

```diff
@@ -0,0 +1,62 @@
+import json
+import os 
+
+from huggingface_hub import create_repo, upload_file
+
+import datasets
+import requests
+
+# API key DeepL to set before running the script with the command 'export DEEPL_API_KEY=***'
+DEEPL_API_KEY= os.environ.get('DEEPL_AUTH_KEY')
+ORGANIZATION = "lyon-nlp/"
+REPO_NAME = "summarization-summeval-fr-p2p"
+SAVE_PATH = "test.json"
+
+summeval = datasets.load_dataset("mteb/summeval")["test"]
+headers = {
+    'Authorization': f"DeepL-Auth-Key {DEEPL_API_KEY}",
+    'Content-Type': 'application/x-www-form-urlencoded',
+}
+data = {
+    'text': "",
+    'target_lang': 'FR',
+}
+trads = []
+
+for line in summeval:
+    trad = line
+    data["text"] = line['text']
+    response = requests.post('https://api.deepl.com/v2/translate', headers=headers, data=data)
+    trad["text"] = response.json()['translations'][0]['text']
+
+    machine_summaries = []
+    for machine_sum in line['machine_summaries']:
+        data["text"] = machine_sum
+        response = requests.post('https://api.deepl.com/v2/translate', headers=headers, data=data)
+        machine_summaries.append(response.json()['translations'][0]['text'])
+    trad['machine_summaries'] = machine_summaries
+
+    human_summaries = []
+    for human_sum in line['human_summaries']:
+        data["text"] = human_sum
+        response = requests.post('https://api.deepl.com/v2/translate', headers=headers, data=data)
+        human_summaries.append(response.json()['translations'][0]['text'])
```

Comment:
Pour éviter la duplication j'aurais fait une fonction qui se charge de traduire et retourner le texte.
`
def translate_with_deepl(text):
    DATA["text"] = text
    response = requests.post('https://api.deepl.com/v2/translate', headers=HEADERS, data=DATA)
    return response.json()['translations'][0]['text']`

Reply:
`data` can be a local variable for your function since it is only used for the post request